great-idea-website/index.html

```diff
@@ -15,49 +15,100 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+	  <header>
+ 	<section class="section1">
+	  <nav>
+	 Services
+  	Product
+  	Vision
+	Features
+  	About
+  	Contact
+	</nav>
+  
 
   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </section>
+  
+	  <section class="section2">
+	  <h1 class="main-header"> 
+ 
+		  <div class="innovation">Innovation</div>    
+		<div class="on">On</div>
+		
+	  	   <div class="Demand">Demand</div>
```

Comment:
Putting `div` on each word is a little much. :sweat_drops: 

But you may ask, "But Ronald, I know all of these words are part of the `h1` tag, but they *need* to be in separate lines!"

No worries, I gotchu fam:
```
<h1 class="main-header">
  Innovation<br>
  On<br>
  Demand
</h1>
```
`<br>` will force a new line, without breaking up semantics or introducing tags on every word!

Reply:
lol! Br looks like a good solution